### DISCUSSION TITLE
Ask for help with more details about the input format

### DISCUSSION THREAD

**Discussion by yingchiyu** [Open-Sn/opensn#205] [Ask for Help] [answered]
Hello! I'd like to inquire about running example scripts. As shown in the image, the executable file for  opensn is located at /home/yyc/xiazai/opensn/build/lua. I attempted to compile the example_test.lua under /home/yyc/xiazai/opensn/test/example, but encountered the issue depicted in the image. Could you please advise me on how to proceed with the input, and will the successful execution generate results in VTK format by default? Thank you for your assistance!
![QQ图片20240407222548](https://github.com/Open-Sn/opensn/assets/116443237/008fdc63-b59f-4e82-9ed5-1fb09b75853c)
![QQ图片20240407222553](https://github.com/Open-Sn/opensn/assets/116443237/5a92112d-1970-4aae-98db-33b639ceed81)

**Comment by andrsd** [comment, marked as answer]
Some tests are build as unit tests. You happen to pick up such a test. These input files can be run with the test binary `opensn/build/test/opensn-test`. 

Other tests do not need the test binary, e.g. `test/modules/cfem_diffusion/c_diffusion_2d_1a_linear.lua` and can be run with the OpenSn binary `opensn/build/lua/opensn`.
